### PR TITLE
very minor atv crate tweaks

### PR DESCRIPTION
**Pull request by Tacolizard**
so there were gonna be headlights added to the atv but i dont feel like fixing the broken headlight code i made so i scrapped it. 

the atv crate has been changed from a plain crate to a large one because realism.
the price has been lowered from 3000 to 2000 #powercreepohno
for memes, the crate includes a leather overcoat and skull bandana so you can make a biker gang
it's been renamed the Biker Gang Kit

**literally how can you say the cost is powercreep when fucking isulated gloves cost as much and aren't contraband and come in an unlocked crate**

**also scooters are also a hard counter to lube and slowdowns so shut the fuck up kevinz**

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
(You could also just put a flashlight in your pocket)

But this is makes sense so 👍

File: config/admins.txt [hunk 1]
```diff
@@ -118,6 +118,7 @@ Aloraydrel = Game Master
 Quiltyquilty = Game Master
 SnipeDragon = Game Master
 Fjeld = Game Master
-kevinz000 = Game Master
+kevinz000 = Game Master //loser
+Tacolizard = Game Master //awesome dude
```
Comment (Mysak0CZ) [inline config/admins.txt]: As far as i know, config and data folders are not copied on the server, during autoupdate...

Comment (Tacolizard) [inline config/admins.txt]: ssshhhhh its a meme you dip

**Comment by Tacolizard** [comment]
ok i might have changed the atv's icon to the flashlight's icon, can somone tell me if i did that?

**Comment by ExcessiveUseOfCobblestone** [comment]
no you made it invisible

File: config/admins.txt (hunk 1 above)
Comment (kevinz000) [inline config/admins.txt]: FUCK YOU.

Comment (kevinz000) [inline config/admins.txt]: FUCK YOU

**Comment by Tacolizard** [comment]
eh i dont feel like coding

**Comment by ExcessiveUseOfCobblestone** [comment]
> Adds items to the crate

>> But Lowers The Cost

**Comment by Tacolizard** [comment]
@ExcessiveUseOfCobblestone cheridan thought it was expensive and so do i. i want it so that making bikergangs isnt super expensive

**Comment by kevinz000** [comment]
2000 point hard counter to lube, and slowdowns.
10/10 :-1:

File: config/admins.txt [hunk 2]
```diff
@@ -118,6 +118,7 @@ Aloraydrel = Game Master
 Quiltyquilty = Game Master
 SnipeDragon = Game Master
 Fjeld = Game Master
-kevinz000 = Game Master
+kevinz000 = Game Master //loser
```
Comment (Jordie0608) [inline config/admins.txt]: This will break loading from this file past this point.

File: config/admins.txt (hunk 1 above)
Comment (optimumtact) [inline config/admins.txt]: remove these comments please

**Comment by Bawhoppen** [comment]
P O W E R   C R EEE P

**Comment by Cheridan** [comment]
@kevinz000 Who cares, R&D can make an autocloner as part of just doing their job as a counter to _being dead._ Even within Cargo these are still more fun and less cancer than meme shotguns.

**Comment by optimumtact** [comment]
TUNNEL SNAKES

**Comment by ExcessiveUseOfCobblestone** [comment]
but lowering the price just allows them to buy both?

I had initially asked for it to be raised to 3000 because every other cutesy item [See Hats, Foam Force, etc.] are above 5000, and have even less utility than the ATVs.

**Comment by Tacolizard** [comment]
@kevinz000 literally one second people complain about too many stuns and the next they object to a counter for stuns

**Comment by PKPenguin321** [comment]
>2000 point hard counter to lube, and slowdowns.

scooters do it for free and they're fine

**Comment by Tacolizard** [comment]
@ExcessiveUseOfCobblestone foam force is only 3000 if I remember correctly and insulated gloves are like 2-3 thousand and come in an unlocked crate

File: config/admins.txt (hunk 1 above)
Comment (Dorsisdwarf) [inline config/admins.txt]: okay

**Comment by Dorsisdwarf** [comment]
Foam Force should only be 800. Tacticool battles every round when

**Comment by ExcessiveUseOfCobblestone** [comment]
@Tacolizard you are correct, although the pistol variant is 4k.

@PKPenguin321 The more you know!

**Pull request merged**